### PR TITLE
docker-image*: add Korean translation

### PR DESCRIPTION
**Pull request by Jueun-Park**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ko/common/docker-images.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
 # docker images
 
-> 도커 이미지를 관리한다.
+> Manage Docker images.
```
Comment (kbdharun) [inline pages.ko/common/docker-images.md:3]: Thanks for your contribution. Can you translate this line?

File: pages.ko/common/docker-images.md [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+`docker images --quiet`
 
-- 어떠한 컨테이너에서도 사용되지 않은 모든 도커 이미지 목록보기:
+- 어떤 컨테이너에도 사용되지 않은 도커 이미지 목록 보기:
 
 `docker images --filter dangling=true`
+
+- 이름에 부분 문자열을 포함한 이미지 목록 보기:
+
+`docker images "{{*부분_문자열*}}"`
```
Comment (kbdharun) [inline pages.ko/common/docker-images.md:25]:
```suggestion
`docker images "{{*이름*}}"`
```
Shouldn't this be `name`?
Feel free to update my suggestion.

Comment (Jueun-Park) [reply pages.ko/common/docker-images.md]: You're right, `name` is `이름`.
I just thought the placeholder might be the `부분_문자열` (`substring`) according to the meaning of the command.
But for consistency of translation, I commit your suggestion. Thanks.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**